jscomp/bsc/rescript_compiler_main.ml

```diff
@@ -106,6 +106,43 @@ let process_file sourcefile ?(kind ) ppf =
   in
   Config.uncurried := uncurried;
   res
+
+let reprint_source_file sourcefile = 
+  Js_config.no_builtin_ppx := true;
```

Comment:
Why does it always have to be true instead of using the existing bsc flag `-bs-no-builtin-ppx`?

Reply:
Good point, I didn't want people to have to know about that flag, but this is an advanced internal command so it's probably useful to be able to switch it on and off. Will remove.